LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-2023 Ghayoor Ul Baqir
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
```

Comment:
- [ ] Great job adding a license agreement to the readme file as requested, however clicking on it still shows a broken link response. Kindly fix this for a more professional readme file.

Checkout the screencast below
[cinnamon-2024-01-10T224208+0200.webm](https://github.com/Xghayor/social-media/assets/81932478/13626e30-ca89-4aae-93a1-3d918019cedd)

Reply:
@Munsa1 Hi,

Thank you for the review. However, could you please verify that when you click on the link, it redirects you to the development branch rather than the PR branch? The work I'm submitting is on another branch, which has been submitted for review. If you switch to the correct branch, it should work fine.